public/app/plugins/datasource/azuremonitor/module.ts

```diff
@@ -12,6 +13,8 @@ export const plugin = new DataSourcePlugin<Datasource, AzureMonitorQuery, AzureM
   .setConfigEditor(ConfigEditor)
   .setQueryEditor(AzureMonitorQueryEditor);
 
+initPluginTranslations(pluginJson.id);
```

Comment:
Not sure if this matters at all but having this first in the file and using an await might prevent any issues?